src/utils/card-formatting.ts

```diff
@@ -85,7 +85,7 @@ export const cardTypes = [
 ] as CardType[];
 
 export const UNKNOWN_CARD_TYPE = {
-  type: 'unknown',
+  type: 'card-unknown',
```

Comment:
updated this so that UnknownCard img element can be imported using `ImageSource[img]` just like other visa/amex card imgs